lib/rust/parser/src/syntax/treebuilding/compound_token.rs

```diff
@@ -260,18 +277,21 @@ impl<'s> CompoundTokenBuilder<'s> for TextLiteralBuilder<'s> {
     }
 
     fn flush(self) -> Option<Tree<'s>> {
-        let Self { open, newline, elements } = self;
+        let Self { open, newline, elements, has_preceding_item: _ } = self;
```

Comment:
A shorter form would be 
```suggestion
        let Self { open, newline, elements, .. } = self;
```
Unless you really want to have an error here when another field will be added.

Reply:
Fields that can correctly be ignored here are the exception.